src/@next/Tabs/Tab.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Typography } from '../Typography';
+import { StyledSpan, StyledTabButton } from './TabStyle';
+
+export type TabProps = {
+  content: React.ReactNode;
+  id?: string;
+  selected?: boolean;
+  onSelect: () => void;
+};
+
+export const Tab = ({ content, id, selected, onSelect }: TabProps) => {
+  return (
+    <StyledTabButton
+      data-selected={selected}
+      id={id}
+      onClick={() => onSelect()}
+    >
+      <StyledSpan>
+        <Typography as="span" variant="body1">
```

Comment:
we might need to let consumer app pass their own content. since the one from Manage Candidate expects to render with a Badge as well. there's a possibility of the need to support other components as well like Icon

Reply:
that would mean there's a Typography wrapper around their own content?

Reply:
Typography as `span` will be rendered as span, and on top of it we get the default font style for the children. So if you passing a text only it'll styled properly as the tab header font, if you passing a component with text (e.g badge) in it also get styled accordingly

Reply:
so basically we need 2 spans to generate the selected "border" style in the bottom